File: Tareas_FdP/Tarea03vscode/Practica_01/a2253320602_Ejercicio02.py
X = float(input("Introduce la longitud de la losa (X): "))
Y = float(input("Introduce el ancho de la losa (Y): "))
N = float(input("Introduce el espesor de la losa (N): "))

volumen = X * Y * N
cemento = volumen * 7
arena = volumen * 0.52
grava = volumen * 0.82
agua = volumen * 200

print("Volumen total de la losa:", volumen, "m3")
print("Cemento necesario (bultos):", cemento)
print("Arena necesaria (m3):", arena)
print("Grava necesaria (m3):", grava)
print("Agua necesaria (litros):", agua)

File: Practicas/Practicas completas/Practica_01/a2253320602_Ejercicio02.cs
using System;

class a2193330234_Practica01_Ejercicio2 {
    static void Main() {
        Console.WriteLine("Introduce la longitud de la losa (X):");
        double X = Convert.ToDouble(Console.ReadLine());
        
        Console.WriteLine("Introduce el ancho de la losa (Y):");
        double Y = Convert.ToDouble(Console.ReadLine());
        
        Console.WriteLine("Introduce el espesor de la losa (N):");
        double N = Convert.ToDouble(Console.ReadLine());
        
        double volumen = X * Y * N;
        double cemento = volumen * 7;
        double arena = volumen * 0.52;
        double grava = volumen * 0.82;
        double agua = volumen * 200;
        
        Console.WriteLine("Volumen total de la losa: " + volumen + " m3");
        Console.WriteLine("Cemento necesario (bultos): " + cemento);
        Console.WriteLine("Arena necesaria (m3): " + arena);
        Console.WriteLine("Grava necesaria (m3): " + grava);
        Console.WriteLine("Agua necesaria (litros): " + agua);
    }
}